internal/process/provisioning/create_runtime_resource_step.go

```diff
@@ -248,9 +244,55 @@ func (s *CreateRuntimeResourceStep) providerValues(operation *internal.Operation
 	return p.Provide(), nil
 }
 
+func (s *CreateRuntimeResourceStep) createHighAvailabilityConfiguration() *gardener.HighAvailability {
+
+	var failureTolerance *gardener.HighAvailability
+	if s.config.ControlPlaneFailureTolerance == string(gardener.FailureToleranceTypeZone) {
+		failureTolerance = &gardener.HighAvailability{
+			FailureTolerance: gardener.FailureTolerance{
+				Type: gardener.FailureToleranceTypeZone,
+			},
+		}
+	} else {
+		failureTolerance = &gardener.HighAvailability{
+			FailureTolerance: gardener.FailureTolerance{
+				Type: gardener.FailureToleranceTypeNode,
+			},
+		}
+	}
+	return failureTolerance
+}
+
+func (s *CreateRuntimeResourceStep) createNetworkingConfiguration(operation internal.Operation) imv1.Networking {
+
+	networkingParams := operation.ProvisioningParameters.Parameters.Networking
+	if networkingParams == nil {
+		networkingParams = &internal.NetworkingDTO{}
+	}
+
+	nodes := networkingParams.NodesCidr
+	if nodes == "" {
+		nodes = networking.DefaultNodesCIDR
+	}
+
+	return imv1.Networking{
+		Pods:     DefaultIfParamNotSet(networking.DefaultPodsCIDR, networkingParams.PodsCidr),
```

Comment:
Is there any case in which DefaultPodsCIDR is not going to be set?

Reply:
I don't get you question.